src/components/skills-quiz/constants.js

```diff
@@ -15,3 +15,15 @@ export const SKILLS_QUIZ_FACET_FILTERS = [
     },
   },
 ];
+
+export const JOBS_QUIZ_FACET_FILTERS = [
```

Comment:
Can you please add this dict in SKILLS_QUIZ_FACET_FILTERS. That way, we don't need SearchJobDropdown.jsx file.

Reply:
@zamanafzal  we want to render SearchJobDropdown if `showSearchJobsAndSearchResults` is set to be `true`.
For that SKILLS_QUIZ_FACET_FILTERS won't be a const anymore?

Reply:
Agreed with Hammad, I tried it this way and it didn't work. The job dropdown doesn't hide when `showSearchJobsAndSearchResults` value is `false`.

Reply:
In that case, I think we don't need to do it this way 
`const filtersFromRefinements = JOBS_QUIZ_FACET_FILTERS.map(({`. Since there will be only one value so we can create a dictionory in constants and pass the values from there.

Reply:
Done